contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py

```diff
@@ -0,0 +1,246 @@
+# with the following modification; you may not use this file except in
+# compliance with the Apache License and the following modification to it:
+# Section 6. Trademarks. is deleted and replaced with:
+#
+# 6. Trademarks. This License does not grant permission to use the trade
+#    names, trademarks, service marks, or product names of the Licensor
+#    and its affiliates, except as required to comply with Section 4(c) of
+#    the License and to reproduce the content of the NOTICE file.
+#
+# You may obtain a copy of the Apache License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
+#
+
+"""Unit tests for the rv reader plugin"""
+
+# Please note that the test is geared towards linux install as I don't have
+# a Windows or Mac available to me.
+
+import os
+import sys
+import ast
+import zipfile
+import tempfile
+import unittest
+import shutil
+import shlex
+import time
+from subprocess import call, Popen, PIPE
+
+import opentimelineio as otio
+
+RV_OTIO_READER_NAME = 'Example OTIO Reader'
+RV_OTIO_READER_VERSION = '1.0'
+
+RV_ROOT_DIR = os.getenv('OTIO_RV_ROOT_DIR', '')
+RV_BIN_DIR = os.path.join(RV_ROOT_DIR, 'bin')
+
+RV_OTIO_READER_DIR = os.path.join(
+    '..',
+    'rv',
+    'example_otio_reader'
+)
+
+# Need this for connecting to RV
+RV_PYTHON_DIR = os.path.join(RV_ROOT_DIR, 'plugins', 'Python')
+if RV_PYTHON_DIR not in sys.path:
+    sys.path.append(RV_PYTHON_DIR)
+
+import rv_tests_network     # noqa: E402
+
+
+# Generate sample data
+sample_timeline = otio.schema.Timeline(
+    'my_timeline',
+    global_start_time=otio.opentime.RationalTime(1, 24)
+)
+track = otio.schema.Track('v1')
+for clipnum in range(1, 4):
+    clip_name = 'clip{n}'.format(n=clipnum)
+    track.append(
+        otio.schema.Clip(
+            clip_name,
+            media_reference=otio.schema.ExternalReference(
+                target_url="{clip_name}.mov".format(clip_name=clip_name),
+                available_range=otio.opentime.TimeRange(
+                    otio.opentime.RationalTime(1, 24),
+                    otio.opentime.RationalTime(50, 24)
+                )
+            ),
+            source_range=otio.opentime.TimeRange(
+                otio.opentime.RationalTime(11, 24),
+                otio.opentime.RationalTime(30, 24)
+            )
+        )
+    )
+sample_timeline.tracks.append(track)
+
+
+@unittest.skipIf(
+    "OTIO_RV_ROOT_DIR" not in os.environ,
+    "OTIO_RV_ROOT_DIR not set."
+)
+@unittest.skipIf(
+    (sys.version_info > (3, 0)),
+    "RV OTIO reader plugin does not work in python 3."
```

Comment:
Who should we add to our list to try the RV beta for when this needs to support RV + Python3?

Reply:
@mikepkes did you get this question answered?  If you ping me on the slack I can route you to someone.